nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/status/ConnectionStatusSnapshotDTO.java

```diff
@@ -48,6 +48,13 @@
     private String queuedCount;
     private Integer percentUseCount;
     private Integer percentUseBytes;
+    private Long predictedMillisUntilCountBackpressure = 0L;
```

Comment:
@markap14 we can do that. Do you see any issues with embedding the BackpressurePredictionDTO inside the ConnectionStatusDTO? We're considering that option because then the data would be available within the same request (less UI/API impact). It would be set to `null` if analytics are disabled.

@YolandaMDavis @mattyb149 @rfellows

Reply:
Seems fair to me @achristianson